src/mgr/ClusterState.h

```diff
@@ -73,6 +76,29 @@ class ClusterState
     return fsmap.get_epoch() > 0;
   }
 
+  bool is_client_in_mgrmap(std::string_view name,
+                           const entity_addrvec_t& client_addr,
+                           epoch_t& e) const {
```

Comment:
```suggestion
                           epoch_t* e) const {
```

outputs are conventionally pointers